tests/test_result_to_dict.py

```diff
@@ -200,6 +202,20 @@ def test_callback(self):
         data = result_to_dict(json_value_without_str_obj, callback=convert_return_none)
         self.assertDictEqual(data, json_value_without_str_obj)
 
+    def test_pymongo_to_dict(self):
+        with connect_to_mongo().get() as client:
+            collection = client.testing_db.example
+            data_not_converted = collection.find()
+            data_converted = result_to_dict(collection.find())
+            sample_entry = {'name': 'rahul'}
+            self.assertIsInstance(data_converted, list)

```

Comment:
please add a line of space between variable definition and assertions. code looks busy